server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -271,19 +273,23 @@ static boolean checkInit(VolumeManager fs, SiteConfiguration sconf, Configuratio
       return false;
     }
     if (sconf.get(Property.INSTANCE_SECRET).equals(Property.INSTANCE_SECRET.getDefaultValue())) {
-      ConsoleReader c = getConsoleReader();
-      c.beep();
-      c.println();
-      c.println();
-      c.println("Warning!!! Your instance secret is still set to the default,"
-          + " this is not secure. We highly recommend you change it.");
-      c.println();
-      c.println();
-      c.println("You can change the instance secret in accumulo by using:");
-      c.println("   bin/accumulo " + org.apache.accumulo.server.util.ChangeSecret.class.getName());
-      c.println("You will also need to edit your secret in your configuration"
-          + " file by adding the property instance.secret to your"
-          + " accumulo.properties. Without this accumulo will not operate" + " correctly");
+      LineReader c = getLineReader();
+      c.getTerminal().puts(InfoCmp.Capability.bell);
+      c.getTerminal().writer().println();
```

Comment:
Does anyone know why the print statements are like this as opposed to one/fewer prints with linebreaks (`\n`). It seems as though there might be a reason `println`s are used in this way, but it seems like it would be cleaner to consolidate this text into less prints if possible. I see that it was this way before this PR so its not quite related but I was just curious.

Reply:
Sometimes this is done for readability, sometimes to reuse a buffer for performance to avoid unnecessary string concatenation (as in StringBuilder append methods), and sometimes to defer to environment-specific implementations (like cross-platform newline differences). It could also just be more convenient for a developer to copy/paste as many `println` calls as they want, rather than edit in a specific number of `\n` in several different String literals.

I'm not sure why it's used this way in this particular case, but probably some combination of readability and developer convenience. But, it probably also makes sense in this case to defer to the terminal's environment-specific implementation, since the terminal itself is intended as an abstraction around the command-line environment.

Reply:
Okay great. Thanks for the explanation.